recipes/master.rb

```diff
@@ -0,0 +1,26 @@
+#
+# Cookbook Name:: osl-jenkins
+# Recipe:: master
+#
+# Copyright 2014, Oregon State University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+node.default['jenkins']['master']['install_method'] = 'war'
```

Comment:
Ah, I was not aware the package _came_ from Jenkins. I assumed it came from the CentOS repos, which would most likely be out of date. Looks like this allows me to swtich the version back to `1.555` as well. Hurray!